bin/install.sh

```diff
@@ -1,286 +1,301 @@
-#!/bin/bash -e
+#!/bin/bash
 
 # vim: tabstop=4 shiftwidth=4 softtabstop=4
 # -*- sh-basic-offset: 4 -*-
 
-WEB_UPGRADE=false
-BRANCH_VERSION=
-MANAGE_NETWORK=
-UPGRADE_SYSTEM=
-
-if [ -f .env ]; then
-    source .env
-fi
-
-while getopts ":w:b:n:s:" arg; do
-    case "${arg}" in
-        w)
-            WEB_UPGRADE=true
-        ;;
-        b)
-            BRANCH_VERSION=${OPTARG}
-        ;;
-        n)
-            MANAGE_NETWORK=${OPTARG}
-        ;;
-        s)
-            UPGRADE_SYSTEM=${OPTARG}
-        ;;
-    esac
-done
-
-if [ "$WEB_UPGRADE" = false ]; then
-
-  # Make sure the command is launched interactive.
-  if ! [ -t 0  ]; then
-    echo -e "Detected old installation command. Please use:\n$ bash <(curl -sL https://install-anthias.srly.io)"
-    exit 1
-  fi
-
-  # clear screen
-  clear;
-
-  # Set color of logo
-  tput setaf 6
-  tput bold
-
-  cat << EOF
-
-       d8888            888     888
-      d88888            888     888       888
-     d88P888            888     888
-    d88P 888  88888b.   888888  88888b.   888   8888b.   .d8888b
-   d88P  888  888 '88b  888     888 '88b  888      '88b  88K
-  d88P   888  888  888  888     888  888  888  .d888888  'Y8888b.
- d8888888888  888  888  Y88b.   888  888  888  888  888       X88
-d88P     888  888  888   Y888   888  888  888  'Y888888   88888P'
-==================================================================
-
-
-EOF
-
-  # Reset color
-  tput sgr 0
-
-  echo -e "Anthias requires a dedicated Raspberry Pi / SD card.\nYou will not be able to use the regular desktop environment once installed.\n"
-  read -p "Do you still want to continue? (y/N)" -n 1 -r -s INSTALL
-  if [ "$INSTALL" != 'y' ]; then
+set -euo pipefail
```

Comment:
@vpetersson, I already set strict mode in this line instead.

Reply:
Fair.